README.md

```diff
@@ -72,8 +72,8 @@ git clone https://github.com/Tencent/TurboTransformers --recursive
 ```
 # You can modify the environment variables in the script to specify the cuda version and operating system version
 sh tools/build_docker_gpu.sh $PWD
-docker run --net=host --rm -it -v $PWD:/workspace -v /etc/passwd:/etc/passwd --name=your_container_name REPOSITORY:TAG
-# for example: docker run --net=host --rm -it -v $PWD:/workspace -v /etc/passwd:/etc/passwd --name=jiarui_gpu_env ccr.ccs.tencentyun.com/mmspr/turbo_transformers:0.1.1-cuda9.0-ubuntu16.04-gpu-dev
+docker run --gpus all --net=host --rm -it -v $PWD:/workspace -v /etc/passwd:/etc/passwd --name=your_container_name REPOSITORY:TAG
```

Comment:
Special reminder.
For multiple-gpu machine, Turbo runs on the GPU of device id 0. In any words, it uses `cudaSetDevice(0)` by default.

Reply:
Thanks for sharing the message.